src/bindings/config.ts

```diff
@@ -90,6 +91,7 @@ export const loadConfig = async (context: Context): Promise<BotConfig> => {
     command: commandSettings,
     assign: {
       bountyHunterMax: bountyHunterMax,
+      staleBountyTime: staleBountyTime ? String(ms(staleBountyTime)) : "0",
```

Comment:
why store it as string if it'll be converted to number later?
`"0"` will never be used because the default config has `30d` so if we want to disable the feature by default than default config should have `0d`

Reply:
Storing it as a string because, ms takes a string input but we need the number later to work with, as default json will have "14d"/"30d" as opposed to a number, I felt it easier to store it as strings and just convert to a number when we need to work with it.

My mistake I read pav's comment as if the default would be that it is active and they'd opt out, but you are right inactive default should be "0d" as opposed to "0".

So should this feature be inactive by default and they'd opt in with their own time range then yeah?

Reply:
I understand now what you meant and don't know why I didn't see it at first, I had a type error and storing as string solved it but obv I was just approaching it wrong, appreciate it foo